modules/sbo-data-model.adoc

```diff
@@ -0,0 +1,70 @@
+// Module included in the following assemblies:
+//
+// *openshift-docs/applications/connecting_applications_to_services/exposing-binding-data-from-a-service
+[id="sbo-data-model_{context}"]
+= Data model
+
+Service binding annotations follow the convention:
+
+[source,yaml]
+----
+service.binding(/<NAME>)?: "<VALUE>|(path=<JSONPATH_TEMPLATE>(,objectType=<OBJECT_TYPE>)?(,elementType=<ELEMENT_TYPE>)?(,sourceKey=<SOURCE_KEY>)?(,sourceValue=<SOURCE_VALUE>)?)"
+----
+where:
+[horizontal]
+`<NAME>`:: Specifies the name under which the binding value is going to be exposed. It can be excluded only when the `objectType` parameter is set to `Secret` or `ConfigMap`.
+`<VALUE>`:: Specifies the constant value exposed, when no `path` is set.
+
+This section explains the data model used in the annotations and OLM descriptors. The data model is the same for custom resource definition (CRD) or custom resource (CR) annotations and Operator Lifecycle Manager (OLM) descriptors, but the syntax is different, which is explained in the respective sections.
```

Comment:
```suggestion
This section explains the data model used in the annotations and OLM descriptors. The data model is the same for custom resource definition (CRD) or custom resource (CR) annotations and Operator Lifecycle Manager (OLM) descriptors, but the syntax differs. The following sections explain [what exactly?].
```
What do the following sections explain? The differences? The similarities? The data model? The annotations? The descriptors? It's not clear to me from the original wording implies that the sections will describe the differences only. (Lmk if you need further clarification on what I mean here.)

Reply:
@bburt-rh The sentence is clearly saying that the syntax is different and that is explained in the sections under [Categories of exposable binding data](https://deploy-preview-39237--osdocs.netlify.app/openshift-enterprise/latest/applications/connecting_applications_to_services/exposing-binding-data-from-a-service#sbo-categories-of-exposable-binding-data_exposing-binding-data-from-a-service). But we cannot use internal links within modules :(

Please suggest a better way to address this? Else, we can remove that sentence and keep "... but the syntax differs". WDYT?

Reply:
I understand. It's another one of those cases where an internal link would be very useful. But since we can't do that, I think it's OK jut to end the sentence with `...but the syntax differs`